### DISCUSSION TITLE
Interactive camera for react-native app

### DISCUSSION THREAD

**Discussion by MykhailoButkevych-MR** [pmndrs/drei#1089] [Q&A]
Hello, I'm new with r3f, how can I setup interactive camera on react-native app? So based on gestures I could rotate camera and zoom in, out?

**Comment by conor-v** [comment]
I think you won't to use the OrbitControls. that let's you zoom in and out and let you rotate around youre scene.
just import from drei and add them inside the canvas.

link --> https://github.com/pmndrs/drei#controls
